code-challenges.js

```diff
@@ -12,53 +12,93 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
-
-
-
+const divisible3 = (num) => {
+    if (num % 3 === 0){
+        return `${num} is divisible by three`
+    } else if (num % 3 === 1 || num % 3 === -1) {
+        return `${num} is not divisible by three`
+    } else {
+        return "error"
+    }
+}
+console.log(divisible3(num1));
+console.log(divisible3(num2));
+console.log(divisible3(num3));
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+const capitalizer = (array) => {
+    for (let i = 0; i < array.length; i++) {
+        array[i] = array[i].charAt(0).toUpperCase() + array[i].substr(1);
+    }
+    return array
+}
 
-
-
+console.log(capitalizer(randomNouns));
 
 // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+const numberFinder = (array) => {
+     let onlyNums = array.filter(value => typeof value === "number")
+     onlyNums.sort((a, b) => a-b)
+     return onlyNums
```

Comment:
you can append these two lines to the previous line if you'd like to save some key strokes! this code works, just a suggestion
ex:  `return array.filter(value => typeof value === "number").sort((a, b) => a-b)`